__tests__/app.test.js

```diff
@@ -235,7 +237,7 @@ describe('Reviews patch request test block', () => {
 })
 
 describe('Users get request testing block', () => {
-    test('Returns all users object in an array with all properties', () => {
+    test('Returns an array of user objects with all properties', () => {
         return request(app).get("/api/users").expect(200).then(res => {
             const {users} = res.body
             expect(users.length).toEqual(4)
```

Comment:
Hi Lukas, I still didn't see the tests for GET /api (should be the endpoints). Can test there're 9 endpoints. Can also test a certain description toBe("<A CERTAIN STRING> ")